src/clout/core.cljc

```diff
@@ -2,24 +2,39 @@
   "A small language for routing."
   (:require [clojure.string :as string]
             [clojure.set :as set]
-            [instaparse.core :as insta]))
+            [instaparse.core :as insta]
+            #?(:cljs [clout.regex :refer [pL]])))
 
 (def ^:private re-chars (set "\\.*+|?()[]{}$^"))
+#?(:cljs (def re-char-keys (seq re-chars)))
+#?(:cljs (def re-char-vals (map #(str "\\" %) re-char-keys)))
 
-(defn- re-escape [s]
-  (string/escape s #(if (re-chars %) (str \\ %))))
+#?(:clj (def ^:private pL "\\p{L}"))
 
-(defn- re-groups* [^java.util.regex.Matcher matcher]
-  (for [i (range (.groupCount matcher))]
-    (.group matcher (int (inc i)))))
+(defn- re-escape [s]
+  #?(:clj
+     (string/escape s #(if (re-chars %) (str \\ %)))
+     :cljs
+     (string/escape s (zipmap re-char-keys re-char-vals))))
+
+#?(:clj
+   (defn- re-groups*
+     [^java.util.regex.Matcher matcher]
+     (for [i (range (.groupCount matcher))]
+       (.group matcher (int (inc i))))))
+
+#?(:cljs
+   (defn re-groups**
+     [re s]
+     (-> (re-seq re s) flatten rest)))
 
 (defn- assoc-conj [m k v]
   (assoc m k
-    (if-let [cur (get m k)]
-      (if (vector? cur)
-        (conj cur v)
-        [cur v])
-      v)))
+           (if-let [cur (get m k)]
```

Comment:
Does this match the rule https://github.com/bbatsov/clojure-style-guide#vertically-align-fn-args right?

Reply:
`assoc` does not have body, so https://github.com/bbatsov/clojure-style-guide#body-indentation rule cannot be applied.

Reply:
Yes, this was indented incorrectly.